activerecord/test/cases/connection_pool_test.rb

```diff
@@ -13,7 +13,7 @@ def setup
 
         # Keep a duplicate pool so we do not bother others
         @db_config = ActiveRecord::Base.connection_pool.db_config
-        @pool_config = ActiveRecord::ConnectionAdapters::PoolConfig.new("primary", @db_config)
+        @pool_config = ActiveRecord::ConnectionAdapters::PoolConfig.new(ActiveRecord::Base, @db_config)
```

Comment:
These should have always been `ActiveRecord::Base` - using primary was deprecated some time ago.